q2_feature_table/plugin_setup.py

```diff
@@ -343,6 +344,9 @@
                        'table instead of being retained.',
         'filter_empty_features': 'If true, features which are not present in '
                                  'any retained samples are dropped.',
+        'allow_empty_table': 'If true, the filtered table may be empty. '
+                             'Otherwise, an error will be raised if the '
```

Comment:
```suggestion
                             'Default behavior is to raise an error if the '
```
Small adjustment just to clarify what the default behavior is!